store-rx2/src/main/kotlin/com/dropbox/store/rx2/RxStore.kt

```diff
@@ -147,3 +150,13 @@ fun <Key : Any, Output : Any, NewOutput : Any> StoreBuilder<Key, Output>.withFlo
         deleteAll = deleteAllFun
     )
 }
+
+/**
+ * Helper factory that will return data as a [Single] for [key] if it is cached otherwise will return fresh/network data (updating your caches)
+ */
+fun <Key : Any, Output : Any> Store<Key, Output>.get(key: Key) = rxSingle { this@get.get(key) }
+
+/**
+ * Helper factory that will return fresh data as a [Single] for [key] while updating your caches
+ */
+fun <Key : Any, Output : Any> Store<Key, Output>.fresh(key: Key) = rxSingle { this@fresh.fresh(key) }
```

Comment:
@digitalbuddha, @yigit what do you think about this name collision? It's not an error because it conflicts with other extension functions (that's part of the core artifact) but it still may be confusing.

On the other hand, if you're using RxJava, calling `getAsSingle` is pretty strange. I'm inclined to keep the simple names - if the user imports the rx artifact they probably want these functions and not the built in one (which they will probably just not import into their file)

Reply:
I'd prefer for optimize imports on the fly to work which won't happen with conflicts. My take is to rename to remove the collision.

Reply:
To clarify your on board for keeping the names as is here (get and fresh)?

Reply:
I'd prefer to make the names unique rather than relying on only one or other to be imported

Reply:
so sounds like you want to request changes?

Reply:
Please rename to `getSingle` and `freshSingle`